cmd/skopeo/utils.go

```diff
@@ -16,8 +17,18 @@ type errorShouldDisplayUsage struct {
 	error
 }
 
+// Check if container-storage are used before doing the maybeReexec
 func needsRexec(c *cli.Context) error {
-	return maybeReexec()
+	for _, arg := range c.Args() {
+		parts := strings.SplitN(arg, ":", 2)
+		if len(parts) != 2 {
+			continue
+		}
+		if parts[0] == storage.Transport.Name() {
```

Comment:
(This could, in principle, be a `transport.RequiresReexecInUserNamespace()` or something like that, but ultimately we are the use of `github.com/containers/`**`storage`**`/pkg/reexec`, so that wouldn’t make the code any more general.)